wiki/Skinning/osu!mania/en.md

```diff
@@ -111,6 +127,96 @@ Notes:
 - This can be disabled in the [options](/wiki/options).
 - Unlike osu!standard and osu!catch's combobursts, all edges of this imageset should not be clipped.
 
+## Keys
+
+`mania-key1.png`
+
+![](img/mania-key1.png)
+
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![No][false] | Normal | Bottom | 50x107 |
+
+Notes:
+
+- This is the idle state.
```

Comment:
i would use `Idle state of the key` and something like this for the ones below 🤷‍♀️